dev/devicelab/lib/framework/browser.dart

```diff
@@ -488,19 +483,19 @@ class BlinkTraceEvent {
   final String ph;
 
   /// Process ID of the process that emitted the event.
-  final int pid;
+  final int? pid;
```

Comment:
These parameters are all required but still nullable?

Reply:
It looks like some changes I meant to go in got lost in the shuffle here. I updated it with my intended changes. They should be non-nullable now.